cmd/gobl/build_test.go

```diff
@@ -341,26 +357,30 @@ func Test_build(t *testing.T) {
 			if opts == nil {
 				opts = &buildOpts{}
 			}
+			opts.indent = true // always indent
 			err := opts.runE(c, tt.args)
 			if tt.err != "" {
 				assert.EqualError(t, err, tt.err)
 			} else {
 				assert.Nil(t, err)
 			}
-			re := testy.Replacement{
+			if tt.replace == nil {
+				tt.replace = make([]testy.Replacement, 0)
+			}
+			tt.replace = append(tt.replace, testy.Replacement{
```

Comment:
I think if you reverse the order here, you can skip the `make()` call above.

i.e:

```
replace := append(testy.Replacement{ ... }, tt.replace...)
```